services/httpd/cursor.md

```diff
@@ -0,0 +1,163 @@
+# Response Cursor Protocol Specification
+
+The cursor protocol is a streaming protocol that may be used by clients
+who are reading from the database using a cursor.
+
+## Encoding
+
+The response structure here is defined independent of any specific
+encoding. The structure here may be encoded using any method of
+encoding whether it be JSON, MessagePack, Protocol Buffers, Flatbuffers,
+etc. The default encoding is MessagePack. While any encoding may be
+used, this document will describe both the general format and describe
+how that format is encoded with MessagePack.
+
+Alternative encodings, if they exist, should use similar data
+structures. Alternative encodings may encode things differently
+taking into account a balance of efficiency and human readability. When
+there is an unclear choice between these, efficiency should take
+priority over human-readability.
+
+All responses will encode each object separately as part of a stream.
+That means every structure in this document is encoded separately in
+a specific order.
+
+## Definitions
+
+Unless otherwise stated, every struct is meant to be encoded as the
+equivalent of a hash map.
+
+## MIME Type
+
+The MIME type for the InfluxDB Cursor Protocol is
```

Comment:
Another possibility for this which might be better is `application/vnd.influxdb-cursor+msgpack; version=1.0`. So the version is a parameter instead of part of the accept header.